### ISSUE TITLE
Changing timeout configuration

### ISSUE THREAD

**Issue by sakhothinskiy** [jose96043/TimezOut#3]
Was trying to allow the user to set the timeout, did not find suiting functionality in this library.
Did I miss something? 
Are there any workarounds to change the timeout?

**Comment by jose96043** [comment]
You mean after setting the timeout once you want to be able to change it later on during the lifecycle of the application?

**Comment by sakhothinskiy** [comment]
Indeed. Like allowing the user to change timeout  in  "Settings" activity.

As i looked into your code I came up with a question: what is the purpose behind "Timer is already initialized" exception when trying to reinitialize Timer?
